docs/release-notes/2.32.0.md

```diff
@@ -0,0 +1,113 @@
+# Percona Monitoring and Management 2.32.0
+
+| **Release date:** | Nov 7, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### State-of-the-art new Home dashboard
+
+In our constant effort to enhance user experience in PMM and considering your feedback, we have added another feather to our cap by replacing the existing home dashboard with a brand new Home dashboard in PMM 2.32.0.  
+
+This new experimental dashboard will be shipped to the PMM users. This dashboard will be uploaded to the Experimental folder to enable the users to test them in their environment and provide feedback.
```

Comment:
@rasika-chivate we aren't uploading it to the Experimental dashboard. This paragraph is not correct :) 
We may say something like this: 
We're releasing this dashboard as your default home dashboard which was released as experimental dashboards in 2.30. Thanks to your feedback, it is now released with an improved version in 2.32.

Reply:
Done